### PR TITLE
BROKER: Added Logging Broker

### PR DESCRIPTION
**Pull request by MahalakshmiMurari**

### CODE REVIEW

**Comment by MahalakshmiMurari** [comment]
Registered Logging Broker

File: ReserveSpot.Services.Api/Brokers/Loggings/LoggingBroker.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+using Microsoft.Extensions.Logging;
+using System;
+
+namespace ReserveSpot.Services.Api.Brokers.Loggings
+{
+    public class LoggingBroker : ILoggingBroker
+    {
+        private readonly ILogger<LoggingBroker> logger;
+
+        public LoggingBroker(ILogger<LoggingBroker> logger)
```
Comment (ShriHumrudha) [inline ReserveSpot.Services.Api/Brokers/Loggings/LoggingBroker.cs]: Single line scope

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]
Fix the conflict

File: ReserveSpot.Services.Api/Brokers/Loggings/LoggingBroker.cs (hunk 1 above)
Comment (MahalakshmiMurari) [reply ReserveSpot.Services.Api/Brokers/Loggings/LoggingBroker.cs]: CODERUB: Modified the single line scope

**Comment by MahalakshmiMurari** [comment]
> 
Resolved the conflict. Please review again

File: ReserveSpot.Services.Api/Brokers/Loggings/LoggingBroker.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+using Microsoft.Extensions.Logging;
+using System;
```
Comment (ShriHumrudha) [inline ReserveSpot.Services.Api/Brokers/Loggings/LoggingBroker.cs:6]: Run code cleanup with the 'Place system directive on top' setting.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: ReserveSpot.Services.Api/Brokers/Loggings/LoggingBroker.cs (hunk 2 above)
Comment (MahalakshmiMurari) [reply ReserveSpot.Services.Api/Brokers/Loggings/LoggingBroker.cs:6]: Done code cleanup with the above option

**Review by ShriHumrudha — APPROVED** [review]

**Review by NaynaMallick — APPROVED** [review]

File: ReserveSpot.Services.Api/Brokers/Loggings/LoggingBroker.cs (hunk 2 above)
Comment (ShriHumrudha) [reply ReserveSpot.Services.Api/Brokers/Loggings/LoggingBroker.cs:6]: I still see the system is not sorted.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**